### ISSUE TITLE
No semicolons

### ISSUE THREAD

**Issue by le717** [Tribex/LDRParser#1]
You don't need semicolons at all. ;)

Wow man, this is nice! :smile: I see some things that could be improved, but overall good job! Honestly, when I get some extra time I will have to contribute back to this. And who knows, maybe I can base the next version of the importer script on this!

**Comment by JoshTheDerf** [comment]
@le717 Haha, yes, I am well aware of the many formatting issues this project has. I didn't have enough time to clean up before I had to go to bed last night. Being primarily a JavaScript developer, there are a great deal of bad habits that carry over to Python.
I will definitely clean up later though, thanks for the input!

And yeah, I wrote this with the importer in-mind, so as to separate the file IO from the parsing of data, which should clear things up quite a bit.
I suppose only Triangles (eHehehEHehEHEheHHEh oh wait...) and quads are needed for the importer script, so a skip-able type list option wouldn't be a bad idea. 
**Edit:** done in 39b154ab27dda6a904b06ff27c8db1f0d90838e9

Be warned though, while this greatly reduces and speeds up file IO, it also uses a considerable amount of RAM for large files.

Hope you enjoy! I might write a WebGL-based demo to test mesh generation, not sure yet.

**Edit:**

---

The concerns of this issue should be addressed in 39b154ab27dda6a904b06ff27c8db1f0d90838e9. Is this close-able?

**Offtopic:**

---

Why is it your avatar reminds me of The Cat in the Hat? Where is it from?
There may be an issue with us working on anything together... You prefer Brackets, and I prefer Atom.  ... Kidding. :)
